rocketpy/Flight.py

```diff
@@ -3584,6 +3574,62 @@ def exportData(self, fileName, *variables, timeStep=None):
 
         return
 
+    # TODO: Add unit tests
+    def exportKML(
+        self, fileName, timeStep=None, template="docs\static\kml_template.xml"
+    ):
+        """Exports flight data to a .kml file, which is supported file for Google Earth
+
+        Parameters
+        ----------
+        fileName : string
+            The file name or path of the exported file. Example: flight_data.csv.
+            Do not use forbidden characters, such as '/' in Linux/Unix and
+            '<, >, :, ", /, \\, | ?, *' in Windows.
+        timeStep : float, optional
+            Time step desired for the data. If None, all integration time steps
+            will be exported. Otherwise, linear interpolation is carried out to
+            calculate values at the desired time steps. Example: 0.001.
+        template : str, optional
+            template kml file containing header of final file, by default "docs\static\kml_template.xml"
```

Comment:
Unfortunately, the files at `docs\static\kml_template.xml` are not installed when the user uses `pip install rocketpy`.

I advise you to learn more about how this can be accomplished here: https://stackoverflow.com/questions/1612733/including-non-python-files-with-setup-py

Reply:
thanks for the simplekml tip, i've implemented it